packages/Babylonian-ExampleMining.package/BPExamplesMorph.extension/instance/initialize.st

```diff
@@ -1,27 +0,0 @@
-*Babylonian-ExampleMining-Core-override
```

Comment:
Several of these example mining methods disappeared. Is this intentional? Where did they go? :)

Reply:
No, will add them back later!